rfcbot.toml

```diff
@@ -1,4 +1,16 @@
-[T-core]
+[fcp_behaviors]
+
+[fcp_behaviors."rust-lang/rfcs"]
+close = true
+postpone = true
+
+[fcp_behaviors."rust-lang/rust"]
+#close = false
```

Comment:
Is this intentionally commented out?

Reply:
This was part of the WIP to test that it doesn't matter functionally; but I'll comment it in for clarity :)